terraform/variables.tf

```diff
@@ -1,51 +1,47 @@
 variable "project_name" {
-    default = "immunisations"
+  default = "immunisations"
 }
 
 variable "project_short_name" {
-    default = "imms"
+  default = "imms"
 }
 
 variable "service" {
-    default = "fhir-api"
-}
-data "aws_vpc" "default" {
-    default = true
-}
-data "aws_subnets" "default" {
-    filter {
-        name   = "vpc-id"
-        values = [data.aws_vpc.default.id]
-    }
+  default = "fhir-api"
 }
 
-locals {
-    root_domain = "${local.config_env}.vds.platform.nhs.uk"
+variable "aws_region" {
+  default = "eu-west-2"
 }
 
 locals {
-    project_domain_name = data.aws_route53_zone.project_zone.name
+  environment       = terraform.workspace == "green" ? "prod" : terraform.workspace == "blue" ? "prod" : terraform.workspace
+  env               = terraform.workspace
+  prefix            = "${var.project_name}-${var.service}-${local.env}"
+  short_prefix      = "${var.project_short_name}-${local.env}"
+  batch_prefix      = "immunisation-batch-${local.env}"
+  config_env        = local.environment == "prod" ? "prod" : "dev"
+  config_bucket_env = local.environment == "prod" ? "prod" : "internal-dev"
+
+  root_domain         = "${local.config_env}.vds.platform.nhs.uk"
+  project_domain_name = data.aws_route53_zone.project_zone.name
+  service_domain_name = "${local.env}.${local.project_domain_name}"
+
+  # For now, only create the config bucket in internal-dev and prod as we only have one Redis instance per account.
```

Comment:
Create ticket for a better fix - if we didn't use a whole Redis instance for config, we could have one bucket per instance of the API